db/migrations/02_add_time_to_peeps_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE peeps ADD COLUMN time VARCHAR(60);
```

Comment:
A better fit would be to use a column of type TIMESTAMP. This has the benefit of the database enforcing data integrity: it will throw an error if anything other than a timestamp is added to that column.